cli/gluster-block.c

```diff
@@ -460,8 +459,8 @@ static int
 glusterBlockModify(int argcount, char **options, int json)
 {
   size_t optind = 1;
-  blockModifyCli mobj = {0, };
-  blockModifySizeCli msobj = {0, };
+  struct blockModifyCli mobj = {{0}, };
```

Comment:
why struct here ? This is typedef already. Same at other places.

Reply:
I checked the whole code, some have the "struct" and some are not, I just make them the same.

Reply:
> @lxbsz
> In most cases the variable names are very big. So lets stick to tydef naming only. Thanks!

Sure, make sense.